### PR TITLE
Feature/seg add old

### PR DESCRIPTION
**Pull request by HechengJin0**
add v1.4.3 segmentation part.

### CODE REVIEW

File: CPAC/pipeline/cpac_pipeline.py [hunk 1]
```diff
@@ -1178,6 +1178,14 @@ def prep_workflow(sub_dict, c, run, pipeline_timing_info=None,
 
             seg_preproc = None
 
+
+            if not any(o in c.seg_use_threshold for o in ["FSL-FAST Thresholding", "Customized Threholding"]):
```
Comment (sgiavasis) [inline CPAC/pipeline/cpac_pipeline.py]: Thresholding (spelling)

File: CPAC/pipeline/cpac_pipeline.py [hunk 2]
```diff
@@ -1178,6 +1178,14 @@ def prep_workflow(sub_dict, c, run, pipeline_timing_info=None,
...
+                err = '\n\n[!] C-PAC says: Your segmentation thresholding options ' \
+                    'setting does not include either \'FSL-FAST Thresholding\' or \'Customized Threholding\'.\n\n' \
```
Comment (sgiavasis) [inline CPAC/pipeline/cpac_pipeline.py]: Here too

File: CPAC/resources/configs/pipeline_config_template.yml [hunk 3]
```diff
@@ -354,7 +354,9 @@ PRIORS_CSF :  $priors_path/avg152T1_csf_bin.nii.gz
 
 
 # Use threshold to further refine the resulting segmentation tissue masks.
-seg_use_threshold : True
+# FSL-FAST Thresholding - Use FSL FAST generated tissue class files (binary masks) to generate the resulting segmentation tissue masks. 
+# Customized Threholding - Set the threshold value for tissue probability maps to generate the resulting segmentation tissue masks.
```
Comment (sgiavasis) [inline CPAC/resources/configs/pipeline_config_template.yml]: And here :)

File: CPAC/seg_preproc/seg_preproc.py [hunk 4]
```diff
@@ -570,7 +576,7 @@ def form_threshold_string(threshold):
             input_1, value_1 = (overlap_segmentmap_with_prior, 'out_file')
 
 
-        if use_threshold:
+        if 'Customized Threholding' in use_threshold:
```
Comment (sgiavasis) [inline CPAC/seg_preproc/seg_preproc.py]: thresholding

File: CPAC/seg_preproc/seg_preproc.py [hunk 5]
```diff
@@ -660,7 +669,7 @@ def form_threshold_string(threshold):
...
```
Comment (sgiavasis) [inline CPAC/seg_preproc/seg_preproc.py]: Thresholding

File: dev/docker_data/default_pipeline.yml [hunk 6]
```diff
@@ -357,7 +357,10 @@ PRIORS_CSF :  $priors_path/avg152T1_csf_bin.nii.gz
 
 
 # Use threshold to further refine the resulting segmentation tissue masks.
-seg_use_threshold : True
+# FSL-FAST Thresholding - Use FSL FAST generated tissue class files (binary masks) to generate the resulting segmentation tissue masks. 
+# Customized Threholding - Set the threshold value for tissue probability maps to generate the resulting segmentation tissue masks.
```
Comment (sgiavasis) [inline dev/docker_data/default_pipeline.yml]: and here

File: CPAC/pipeline/cpac_pipeline.py (hunk 1 above)
Comment (HechengJin0) [reply CPAC/pipeline/cpac_pipeline.py]: kk. updated.

**Pull request merged**